code/Player/Corpse.cs

```diff
@@ -213,7 +213,7 @@ public void SendKillInfo( To to )
 
 			Player.SendDamageInfo( To.Single( client ) );
 
-			SendMiscInfo( To.Single( client ), KillList, Perks, C4Note, TimeUntilDNADecay );
+			SendMiscInfo( To.Single( client ), KillList, Perks, C4Note, Player.LastWords, TimeUntilDNADecay );
```

Comment:
Since we are accessing the `Player.LastWords`, the `_timeSinceLastWords` has always been ticking. Wouldn't we need to store it onto the corpse to confirm that they made the message in the last three seconds of them being alive?